index.html

```diff
@@ -21,6 +21,9 @@ <h1 class="header">TODO List</h1>
         <button id="add-todo">
           <span class="material-symbols-outlined addIcon"> add </span>
         </button>
+        <button id="edit-todo" style="display: none;">
```

Comment:
@RiyaBasak26 confusing id the button is saying add but the id assigned is edit-todo